cypress/support/commands.ts

```diff
@@ -1,4 +1,11 @@
-import type { ExceptionEvent, LogEvent, MeasurementEvent, TraceEvent, TransportBody } from '@grafana/agent-core';
+import type {
+  ExceptionEvent,
+  LogEvent,
+  MeasurementEvent,
+  TraceEvent,
+  TransportBody,
+  EventEvent,
```

Comment:
This will fail with latest main branch (it contains a rule to enforce member ordering in imports - also contains autofix).